tests/release-gate/dogfood-examples/delta/rc11/scripts/hooks/tech-lead-authoring-guard.py

```
@@ -0,0 +1,841 @@
+    return False
+
+
+def _is_customer_notes_path(path: str) -> bool:
+    norm = _normalise(path)
+    if not norm:
+        return False
+    if os.path.basename(norm) == CUSTOMER_NOTES_BASENAME:
+        return True
+    if norm.startswith(CUSTOMER_NOTES_DIR_PREFIX):
+        return True
+    return False
```

Ruff (0.15.12)

[warning] Return the condition `bool(norm.startswith(CUSTOMER_NOTES_DIR_PREFIX))` directly

Replace with `return bool(norm.startswith(CUSTOMER_NOTES_DIR_PREFIX))`

(SIM103)